ui/app/utils/supervised_fine_tuning/openAITokenCounter.test.ts

```diff
@@ -24,7 +24,7 @@ const shouldRunApiTests =
   process.env.ENABLE_TEST_WITH_OPENAI_API === "true";
 
 const client = new OpenAI({
-  apiKey: process.env.OPENAI_API_KEY,
+  apiKey: process.env.OPENAI_API_KEY || "fake-api-key",
```

Comment:
it is preferable to use a `.env.development` file (and throwing in the code if the var is not present) instead of hard coding default development env variables

Reply:
Thanks - managed to get this to work without it :)